sdk/lib/opentelemetry/sdk/resources/resource.rb

```diff
@@ -31,11 +31,22 @@ def create(labels = {})
           end
 
           def telemetry_sdk
-            create(
+            resource_labels = {
               Constants::TELEMETRY_SDK_RESOURCE[:name] => 'opentelemetry',
               Constants::TELEMETRY_SDK_RESOURCE[:language] => 'ruby',
               Constants::TELEMETRY_SDK_RESOURCE[:version] => OpenTelemetry::SDK::VERSION
-            )
+            }
+
+            resource_pairs = ENV['OTEL_RESOURCE_ATTRIBUTES']
```

Comment:
I think it makes sense to have this baked into the default resource that gets created during configuration.